packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.10.2",
+  "version": "7.0.0-alpha.0",
```

Comment:
I did include this in the major just to keep it in the same major version range

Reply:
This was necessary anyways, because of the #10924 😉